cypress.config.js

```diff
@@ -2,8 +2,10 @@ const { defineConfig } = require('cypress');
 
 module.exports = defineConfig({
   e2e: {
+    viewportHeight: 1080,
+    viewportWidth: 1920,
     setupNodeEvents(on, config) {
       // implement node event listeners here
     }
   }
-});
+});
\ No newline at end of file
```

Comment:
Add a newline at the end of the file to maintain consistency and avoid potential issues with some tools.